src/scripts/main.js

```diff
@@ -1,3 +1,205 @@
 'use strict';
 
-// write your code here
+const placeColums = [[], [], [], []];
+const buttonPlay = document.querySelector('.button');
+const BASE_CELL_CLASS = 'field-cell';
+const score = document.querySelector('.game-score');
+const gameTable = document.querySelector('.game-header');
+const plaseRow = [...document.querySelector('tbody').children]
+  .map(element => [...element.children]);
+
+for (let i = 0; i < 4; i++) {
+  for (let k = 0; k < 4; k++) {
+    placeColums[k].push(plaseRow[i][k]);
+  }
+}
+
+gameTable.dataset.value = 'off';
+
+function randomInteger(min, max) {
+  const rand = min - 0.5 + Math.random() * (max - min + 1);
+
+  return Math.round(rand);
+}
+
+function FirtsValue(arr) {
+  const a = randomInteger(0, arr.length - 1);
+  const valueA = randomInteger(1, 10) === 10 ? 4 : 2;
+
+  arr[a].innerHTML = `${valueA}`;
+  arr[a].classList.add(`${BASE_CELL_CLASS}--${valueA}`);
+  arr.splice(a, 1);
+}
+
+function restartMesagge() {
```

Comment:
Consider renaming 'restartMesagge' to 'restartMessage' for better readability and proper spelling.